### DISCUSSION TITLE
Print random docs on random printer of my Rustdesk network

### DISCUSSION THREAD

**Discussion by stereosoul** [rustdesk/rustdesk#4455] [General]
### Bug Description

Greetings. I use the Rustdesk selfhost option and clients are installed in completely different locations. Today a strange incident occurred, at one of the locations in the morning a bundle of documents was printed, consisting of documents from completely different organizations and the only thing that connects them is that I installed Rustdesk on their workplaces and configured on your server. How can it be?

### How to Reproduce

idk

### Expected Behavior

Documents should only be printed where they are supposed to be printed.

### Operating system(s) on local side and remote side

Windows 10

### RustDesk Version(s) on local side and remote side

1.1.9

### Screenshots

-

### Additional Context

-

**Comment by stereosoul** [comment]
How can I diagnose this issue? There are very few logs on clients.

**Comment by rustdesk** [comment]
Sadly to hear this. But, frankly, totally no clue, this is beyond our knowledge. So far we have nothing to do with printer, though a lot of users ask for remote printer support.

**Comment by stereosoul** [comment]
Are there print job logs on the server side? In general, Rustdesk doesn't forward local printers remotely like RDP does, right? Potentially this is a serious security issue.

**Comment by stereosoul** [comment]
I figured out the incident, this is not a RustDesk bug, thanks for the answer
